modules/check_in/spec/workers/travel_claim_submission_worker_spec.rb

```diff
@@ -159,5 +164,164 @@
         end
       end
     end
+
```

Comment:
I don't think we need to recreate every spec for when the feature flag is enabled. One spec should be sufficient.